contactaccessrights/js/test/controllers/access-rights-modal-ctrl_test.js

```diff
@@ -222,6 +222,45 @@ define([
             expect(ctrl.errorMsg.length).not.toBe(0);
           });
         });
+
+        describe("when location changes", function () {
+          it("cannot submit without change", function () {
+            expect(ctrl.canSubmit).toBe(false);
+          });
+
+          it("can submit when location and region changes", function () {
+
+            ctrl.selectedData.locations.push({
```

Comment:
You are not testing when either of `vm.selectedData.locations.length` or `vm.selectedData.regions.length` is false